sql/core/src/test/resources/sql-tests/results/cast.sql.out

```diff
@@ -458,3 +458,59 @@ struct<CAST(
  xyz 	
 AS BOOLEAN):boolean>
 -- !query output
 NULL
+
+
+-- !query
+SELECT CAST('1 year 2 month' AS INTERVAL YEAR TO MONTH)
+-- !query schema
+struct<CAST(1 year 2 month AS INTERVAL YEAR TO MONTH):interval year to month>
+-- !query output
+1-2
+
+
+-- !query
+SELECT CAST('1 year 2 month 3 day' AS INTERVAL YEAR TO MONTH)
+-- !query schema
+struct<CAST(1 year 2 month 3 day AS INTERVAL YEAR TO MONTH):interval year to month>
+-- !query output
+1-2
```

Comment:
Not sure that we should support casting of mixed interval as ANSI intervals. cc @cloud-fan

Reply:
@MaxGekk Do I need to continue this PR？